MDANSE_GUI/Src/MDANSE_GUI/Tabs/Models/PlotDataModel.py

```diff
@@ -185,6 +208,22 @@ def add_file(self, filename: str):
             self._next_number += 1
             self.appendRow(new_item)
             new_item.populate(new_datafile._file)
+            self.finished_loading.emit(self._next_number - 1)
+            self.recent_files_update.emit(filename)
```

Comment:
I think it would make more sense to move the `self._next_number` to the end of the block rather than doing a `-1`